README.md

```diff
@@ -1,2 +1,16 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable running machine learning infrastructure on top of Kubernetes.
+The primary focus of MLKube is:
+
+- Resource Management
+- Distributed Workloads Scheduling
```

Comment:
I am a little confused, Do you mean ML workloads?

Reply:
Yes

Reply:
Could we use ML instead of distributed since they have some differences :thinking:

Reply:
Update to `Distributed ML Workloads Scheduling`